### PR TITLE
Extend OpenSCAP section

### PR DESCRIPTION
**Pull request by maximiliankolb**
This PR adds a procedure to install OpenSCAP plugin, to run OpenSCAP scans, and a Hammer CLI procedure to upload extra SCAP content.

### CODE REVIEW

File: guides/doc-Administering_Red_Hat_Satellite/topics/Security_Compliance_Management.adoc [hunk 1]
```diff
@@ -19,7 +19,7 @@ include::xccdf-profile.adoc[leveloffset=+3]
 
 include::listing-available-xccdf-profiles.adoc[leveloffset=+4]
 
-
+include::../common/modules/proc_installing-openscap-plugin.adoc[leveloffset=+2]
```
[the remark this answers is not in the document]
Comment (lzap) [reply guides/doc-Administering_Red_Hat_Satellite/topics/Security_Compliance_Management.adoc:22]: Satellite: yes. Foreman: no.

[the remark this answers is not in the document]
Comment (lzap) [reply guides/doc-Administering_Red_Hat_Satellite/topics/Security_Compliance_Management.adoc:22]: In other words: this change is correct.

File: guides/doc-Administering_Red_Hat_Satellite/topics/security-content-automation-protocol.adoc [hunk 2]
```diff
@@ -1,9 +1,9 @@
-[id='security-content-automation-protocol_{context}']
+[id="security-content-automation-protocol_{context}"]
```
Comment (lzap) [inline guides/doc-Administering_Red_Hat_Satellite/topics/security-content-automation-protocol.adoc:1]: Out of my curiousity - does this change anything or is it just purely cosmetic?

File: guides/doc-Administering_Red_Hat_Satellite/topics/uploading-extra-scap-content.adoc [hunk 3]
```diff
@@ -1,15 +1,30 @@
-[id='uploading-extra-scap-content_{context}']
+[id="Uploading_Extra_SCAP_Content_{context}"]
```
Comment (lzap) [inline guides/doc-Administering_Red_Hat_Satellite/topics/uploading-extra-scap-content.adoc:1]: I personally prefer `this-id-convention` but I am good. We will probably never be able to fix everything.

**Review by lzap — APPROVED** [review]
No issues, thanks.

File: guides/doc-Administering_Red_Hat_Satellite/topics/security-content-automation-protocol.adoc (hunk 2 above)
Comment (maximiliankolb) [reply guides/doc-Administering_Red_Hat_Satellite/topics/security-content-automation-protocol.adoc:1]: IMHO purely cosmetic/ for convenience.

File: guides/doc-Administering_Red_Hat_Satellite/topics/uploading-extra-scap-content.adoc (hunk 3 above)
Comment (maximiliankolb) [reply guides/doc-Administering_Red_Hat_Satellite/topics/uploading-extra-scap-content.adoc:1]: Well, don't underestimate my love for consistency 😂  Only 118 instances to go.

Comment (lzap) [reply guides/doc-Administering_Red_Hat_Satellite/topics/uploading-extra-scap-content.adoc:1]: I wrote this BEFORE I saw part1 and part2 anchors PRs :-)

**Pull request merged**